play/src/components/FooterContent.tsx

```diff
@@ -1,13 +1,8 @@
 import React from 'react';
 import { styled, useTheme } from '@mui/material/styles';
 import { Typography, Container } from '@mui/material';
+import { monsterMap } from '../lib/PlayModels';
```

Comment:
These changes are in anticipation of [[Play] - Select Avatar Page #590](https://github.com/rightoneducation/righton-app/pull/590). They move the icon handling to lib